src/AstTranslator.h

```diff
@@ -59,27 +58,27 @@ class AstTranslator {
     const TypeEnvironment* typeEnv;
 
     /**
-     * The location of some value in a loop nest.
+     * Concrete attribute
      */
     struct Location {
         int level;         // < the loop level
-        int component;     // < the component within the tuple created in the given level
+        int element;       // < the element within the tuple created in the given level
```

Comment:
While you're modifying this, could you remove the '< ' from these three comments? We don't use it anywhere else.